src/crimson/tools/objectstore/objectstore_tool.h

```diff
@@ -0,0 +1,114 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+#include <optional>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "common/hobject.h"
+#include "crimson/os/alienstore/alien_store.h"
+#include "crimson/os/futurized_store.h"
+#include "include/expected.hpp"
+#include "osd/osd_types.h"
+#include "seastar/core/future.hh"
+
+using crimson::os::FuturizedStore;
+
+namespace crimson {
+namespace tools {
+namespace kvstore {
+
+extern seastar::logger logger;
```

Comment:
Let's avoid global variable in header if possible.

Probably get it from `StoreTool` for `crimson_objectstore_tool.cc`, making sure it is only available in the seastar world.

Reply:
I don't see a reason to use seastar logger for the tool operations, unless I'm missing anything.
I've removed the logger above and switched all tool logs to fmt::print (cout) instead. Seastore's (seastar logger) logs are would still be available when passing `--debug`.